backend/src/main/java/com/redhat/cloud/notifications/db/EndpointEmailSubscriptionResources.java

```diff
@@ -15,77 +15,68 @@
     @Inject
     Mutiny.SessionFactory sessionFactory;
 
-    public Uni<Boolean> subscribe(String accountNumber, String username, String bundleName, String applicationName, EmailSubscriptionType subscriptionType) {
-        String query = "INSERT INTO endpoint_email_subscriptions(account_id, user_id, application_id, subscription_type) " +
-                "SELECT :accountId, :userId, a.id, :subscriptionType " +
-                "FROM applications a, bundles b WHERE a.bundle_id = b.id AND a.name = :applicationName AND b.name = :bundleName " +
-                "ON CONFLICT (account_id, user_id, application_id, subscription_type) DO NOTHING"; // The value is already on the database, this is OK
+    public Uni<Boolean> subscribe(String accountNumber, String username, String bundleName, String applicationName,
+            EmailSubscriptionType subscriptionType) {
+        String query = "INSERT INTO endpoint_email_subscriptions(account_id, user_id, application_id, subscription_type) "
+                + "SELECT :accountId, :userId, a.id, :subscriptionType "
+                + "FROM applications a, bundles b WHERE a.bundle_id = b.id AND a.name = :applicationName AND b.name = :bundleName "
+                + "ON CONFLICT (account_id, user_id, application_id, subscription_type) DO NOTHING"; // The value is
+                                                                                                     // already on the
+                                                                                                     // database, this
+                                                                                                     // is OK
```

Comment:
That kind of thing is also bad. The comment only applies to line 23. We should be able to write both the line and the full comment in a single line.